frontend/src/components/StageNotes.tsx

```diff
@@ -0,0 +1,37 @@
+import { PopulatedReview } from "../api";
+import { formatFieldNameHumanReadable } from "../helpers/review";
+
+interface StageNotesProps {
+  reviewsInStage: PopulatedReview[];
+}
+
+/* Lists notes from all reviewers */
+export default function StageNotes({ reviewsInStage: reviews }: StageNotesProps) {
+  const notesFields = reviews.flatMap((r) =>
+    Object.entries(r.fields)
+      .filter(([key, _]) => key.includes("notes"))
+      .map(([key, value]) => [key, value, r.reviewerEmail] as [string, string, string])
+  );
+
+  if (notesFields.length === 0) return null;
+
+  return (
+    <div className="tw:bg-cream-primary tw:p-6 tw:rounded-lg">
+      <h3 className="tw:!m-0 tw:!mb-2 tw:!text-teal-primary tw:!text-xl">Notes: </h3>
+      <div className="tw:flex tw:flex-col tw:gap-3">
+        {notesFields.map(([key, value, email]) => {
+          const label = formatFieldNameHumanReadable(key).replace("Notes", "").trim();
+
+          return (
+            <div key={`${key}-${email}`} className="tw:flex tw:flex-col">
+              ({email})
+              <div>
+                <span className="tw:font-medium">{label === "" ? "General" : label}</span>: {value}
```

Comment:
Looks like line breaks aren't rendered correctly in the notes, can you fix? e.g. application `670cb00f47d79b7ca00a8941` in 2024 DB

<img width="1516" height="117" alt="Screenshot from 2025-09-14 20-25-13" src="https://github.com/user-attachments/assets/5ecee9d7-6894-47cc-930f-591b05fb4e1b" />